.github/workflows/build.yaml

```
@@ -74,7 +74,13 @@ jobs:
                 export runs_on="ubuntu-latest"
                 export platform=""
                 export extra_pytest=""
-                export build_type="dev"
+                if [[ "${GITHUB_REF}" == "refs/tags/*" ]]; then 
+                  export build_type="release"; 
+                  export target="nil"; 
```

Can't we use empty target here?
```suggestion
                  export target=""; 
```